src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -18,6 +18,77 @@ paths:
   # --------------------------------------------------------------------------------
   #
 
+  #
+  # Pre-prison Education
+  # --------------------------------------------------------------------------------
+  '/person/{prisonNumber}/pre-prison-education':
```

Comment:
I'm not 100% sold on the name pre-prison-education although I can't quite put my finger on why it doesn't feel right. Perhaps because some of it may have been delivered on previous visits to prison? The terminology used in the new designs is "educational qualifications history" which could be shortened to education-history maybe?

It might just be an artefact of having "education qualifications" mastered by two different services when in reality if I'm preparing someone for work I only care if a person has a Maths GCSE or not and not whether it was earned in prison or not (although showing development in prison might also be a factor) 🤷‍♂️

Reply:
Yeah, that's a fair challenge 👍 
I've asked the question on the plp-dev-ba channel; hopefully Aliki or Ben will be able to provide some clarity on this 👍